### DISCUSSION TITLE
how can i Manually fixing data

### DISCUSSION THREAD

**Discussion by DFOUD** [teslamate-org/teslamate#3012] [General]
![image](https://user-images.githubusercontent.com/44726767/206266330-7afae8bd-9aea-453d-80fe-0223fc7ebd21.png)
How do I fix this incorrect charging record, I can't possibly charge 400 times in just a few days
Thanks for the reply

**Comment by CrowMash** [comment]
Can you see the 471 charging processes in the Charges dashboard ? It could be a good start to see what is wrong.
